server/services/coastline/seed_coastline.py

```diff
@@ -1,5 +1,5 @@
 from server.crud.crud_coastline import coastline
-from server.schemas import CoastlineCreate
+from server.schemas.coastline import CoastlineCreate
```

Comment:
Why the import change?

Reply:
I think some of this may be the work of your scripts. Either that, or I'm implicitly following absolute path convention. I'm not 100% sure. If it's not your scripts then it's just me following patterns and conventions like before.

Reply:
I don't think my scripts add that. 

To test, delete the "coastline" part of the import and run `poetry run lint` and/or `pre-commit run --all-files`. Let me know if it gets added back after running those commands and that'll confirm it.

> If it's not your scripts then it's just me following patterns and conventions like before.

See my comment about pathing conventions [here](https://github.com/clairBuoyant/server/pull/44#discussion_r959197667). I think you're mixing up what relative and absolute imports are. Let me know if anything is unclear or would like to discuss further.